src/utils/validation.ts

```diff
@@ -1,29 +1,42 @@
-import { isEmpty, pipe, filter, isNil } from 'ramda';
+import {
+  isEmpty,
+  pipe,
+  filter,
+  isNil,
+  gt,
+  lt,
+  gte,
+  lte,
+  find
+} from 'ramda';
 
-export const isValid = (...validators): any => {
-  return (arg: any) => {
-    // TODO: Find appropriate function for this...
-    for (const val of validators) {
-      if (!val(arg)) {
-        return false;
-      }
-    }
-    return true;
-  };
-};
+export const isValid = (...validators): any =>
+  (arg: any) => isNil(find(val => !val(arg), validators));
 
 // Validator functions
 export const maxNumberValidation = (max: number) =>
-  (value: number) => value < max;
+  (value: number) => lt(value, max);
+
+export const minNumberValidation = (min: number) =>
+  (value: number) => gt(value, min);
+
 export const maxStringLengthValidation = (max: number) =>
-  (value: string) => value.length < max;
+  (value: string) => value
+    ? maxNumberValidation(max)(value.length)
+    : true;
+
 export const minStringLengthValidation = (min: number) =>
-  (value: string) => value.length > min;
-export const arrayNotEmptyValidation = () =>
-  (value: string[]) => {
-    const hasValue = pipe(
-      filter(isNil),
-      isEmpty
-    );
-    return isEmpty(value) ? false : hasValue(value);
-  };
+  (value: string) => value
+    ? minNumberValidation(min)(value.length)
+    : true;
```

Comment:
had to add this here because empty form was throwing errors trying to access length from undefined
we got to remember testing the form creating a new wizard
maybe we could have a test with a form in a blank state expect the validation to be falsy, just to make sure nothing will blow up